src/plugins/intel_cpu/src/nodes/interaction.cpp

```diff
@@ -2,27 +2,161 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
-#include <algorithm>
 #include <chrono>
-#include <cmath>
 #include <string>
 #include <vector>
 
 #include "ngraph_transformations/op/interaction.hpp"
 #include "interaction.h"
-#include "utils/general_utils.h"
 #include <onednn/dnnl.h>
 #include <dnnl_extension_utils.h>
 #include "nodes/common/cpu_convert.h"
 #include "memory_desc/cpu_memory_desc_utils.h"
 #include "memory_desc/dnnl_blocked_memory_desc.h"
 #include "common/bfloat16.hpp"
 #include "common/cpu_memcpy.h"
+#include <ie_ngraph_utils.hpp>
 #include <cpu/x64/cpu_isa_traits.hpp>
+#include <cpu/x64/jit_generator.hpp>
+#include "emitters/jit_dnnl_emitters.hpp"
+#include "emitters/jit_load_store_emitters.hpp"
 
 namespace ov {
 namespace intel_cpu {
 namespace node {
+using namespace Xbyak;
+#define THROW_ERROR IE_THROW() << getTypeStr() << " node with name '" << getName() << "' "
+template <cpu_isa_t isa>
+struct jit_move_scale_kernel : public jit_uni_move_scale_kernel, public jit_generator {
+    DECLARE_CPU_JIT_AUX_FUNCTIONS(jit_move_scale_kernel)
+
+    explicit jit_move_scale_kernel(const jit_move_scale_compile_params& jcp) : jit_uni_move_scale_kernel(jcp), jit_generator(jit_name()) {
+        runtime_prc = jcp_.src_prc == Precision::BF16 ? Precision::BF16 : Precision::FP32;
+        if (jcp_.dst_prc == Precision::I8 || jcp_.dst_prc == Precision::U8)
+            runtime_prc = Precision::FP32;
+        if (runtime_prc == Precision::BF16)
+            vec_size = dnnl::impl::cpu::x64::cpu_isa_traits<isa>::vlen / sizeof(uint16_t);
```

Comment:
`vec_size = dnnl::impl::cpu::x64::cpu_isa_traits<isa>::vlen / runtime_prc.size(); `?

Reply:
fix in 48348011fbb78d098132446478cefb3bad0e44c8 thx!